front/src/modules/people/services/select.ts

```diff
@@ -43,6 +47,28 @@ export function usePeopleQuery(
   });
 }
 
+export function useFilteredSearchPeopleQuery(
+  searchFilter: string,
```

Comment:
could you move these 3 parameters to an object: it improves the dev experience while using the function:
{ searchFilter: string, selectedPeopleIds: Array<string>, limit?: number }